pm_icecon/land_spillover.py

```diff
@@ -30,86 +15,15 @@
           mock concentration value is greater than the calculated conc value
 """
 
-
-def read_adj123_file(
```

Comment:
These files and how we manage them is specific to the CDR, so I've extracted them to the `seaice_ecdr` for now.